pydruid/db/api.py

```diff
@@ -222,18 +206,16 @@ def __init__(
         user=None,
         password=None,
         context=None,
-        header=None,
+        header=False,
         ssl_verify_cert=True,
         proxies=None,
         ssl_client_cert=None,
     ):
-        if header is not None:
+        if header is not True:
```

Comment:
I would mantain the default as None, and only warn when an explicit header=False is passed. Otherwise, clearly the caller doesn't care and the lib can "just work"